tokio/src/runtime/builder.rs

```diff
@@ -75,6 +71,12 @@ enum Kind {
     ThreadPool,
 }
 
+#[cfg(not(loom))]
+type Callback = Arc<dyn Fn() + Send + Sync>;
+
+#[cfg(loom)]
+type Callback = Arc<Box<dyn Fn() + Send + Sync>>;
```

Comment:
Why is the box required with `loom`?

Reply:
Loom doesn't support trait objs... partially because it isn't really doable on stable.

Reply:
aha